### PR TITLE
Update Elixir on GCE tutorial to cover Phoenix 1.4 and Cloud SQL

### LINKED ISSUE #582: Issue with elixir-phoenix-on-google-compute-engine
The first step of "Create and configure a Compute Engine instance" fails because **debian-8** is not available - only **debian-9**.

### PR DESCRIPTION
**Pull request by dazuma**
This is a long-needed update to the Elixir Phoenix on GCE tutorial.

It updates the tutorial to cover recent updates to the Phoenix framework (version 1.4) and the Distillery build tool (version 2.0), as well as updates to the Docker base images and GCE images. More significantly, it now includes material on how to connect to a Postgres database running in Cloud SQL, a very common request from readers.

See also the corresponding updates to the Elixir Phoenix on GAE tutorial in #605 and the GKE tutorial in #622

Should fix #582

### CODE REVIEW

File: tutorials/elixir-phoenix-on-google-compute-engine/index.md [hunk 1]
```diff
@@ -62,54 +67,246 @@ tasks on your workstation:
...
 
-3.  Install **Elixir 1.4 or later** if you do not already have it. If you are
-    on MacOS and have [Homebrew](https://brew.sh), you can run
+    Version 227.0.0 or later of the SDK is required. If you have an earlier
+    version installed, you may upgrade it by running:
+
+        gcloud components update
+
+3.  Install **Elixir 1.5 or later** if you do not already have it. If you are
+    on MacOS and have [Homebrew](https://brew.sh), you can run:
```
Comment (ToddKopriva) [inline tutorials/elixir-phoenix-on-google-compute-engine/index.md]: Apple has switched to 'macOS' as the official name of the OS.

File: tutorials/elixir-phoenix-on-google-compute-engine/index.md [hunk 2]
```diff
@@ -164,32 +365,37 @@ Now you can create a release to test out your configuration.
...
 ## Building a release for deployment
 
 Building a release for deploying to the cloud is a bit more complicated
 because your build needs to take place in the same operating system and
 architecture as your deployment environment. In this section, you will use
-Docker to cross-compile a release (which will eventually run directly on a
-Compute Engine VM, not in a Docker container). Then, you will upload your
-release to a Cloud Storage bucket.
+Docker to cross-compile a release. (However, your app will eventually run
+directly on a Compute Engine VM, not in a Docker container). Then, you will
```
Comment (ToddKopriva) [inline tutorials/elixir-phoenix-on-google-compute-engine/index.md]: I'd put the period inside the parentheses in this case.

**Comment by ToddKopriva** [comment]
- [ ] editorial review: @ToddKopriva 
- [ ] production pass: @ToddKopriva

**Comment by dazuma** [comment]
@ToddKopriva Regarding the HTML tags, would using html escapes (`&lt;` and `&gt;`) help? How has this been done in other tutorials? I can't imagine this is the first time this has come up.

**Comment by dazuma** [comment]
@ToddKopriva It looks like using `&lt;` and `&gt;` may do the right thing. See line 249 of https://github.com/GoogleCloudPlatform/community/blob/master/tutorials/appengine-pusher/index.md which includes those codes, and then note how it is rendered in production (https://cloud.google.com/community/tutorials/appengine-pusher). The `<strong>` and `</strong>` tags get dropped by the renderer, but the `&lt;` and `&gt;` are unescaped and rendered as `<` and `>` characters.

(Incidentally, it looks like that other tutorial needs to be fixed; the intent appears for those codes to appear literally in the sample code. Actually, doing a search for html tags throughout the tutorials yields a bunch of cases of apparent issues in existing markdown files. I can open an issue with a dump of those if you want.)

**Comment by ToddKopriva** [comment]
In cases of lengthy HTML code examples, we recommend pulling the code into a separate file that we can then include by reference.

In the case of short snippets, I think that the best approach may be to use the & entities that you mention. I just haven't tested this yet.

Yes, if you have seen cases where HTML items are messed up in existing tutorials, please let me know the details. You can do so in email, rather than through this GitHub comment thread. Thanks.

**Comment by dazuma** [comment]
Okau, pushed a commit to escape the angle brackets using html entities.

**Comment by dazuma** [comment]
Removed the angle brackets since they aren't strictly necessary.

**Comment by ToddKopriva** [comment]
Thanks, Daniel. I'll do the production pass today.

**Pull request merged**